src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -255,6 +255,34 @@ void ValidateContent(string expectedContent, string actualContent)
                 }
             }),
 
+            ("POST Multipart Data",
+            async ctx =>
+            {
+                (string expected, MultipartContent formDataContent) formData = GetMultipartContent(contentSource, ctx, numParameters);
+                Version httpVersion = ctx.GetRandomVersion(httpVersions);
+
+                using (var req = new HttpRequestMessage(HttpMethod.Post, serverUri) { Version = httpVersion, Content = formData.formDataContent })
+                using (HttpResponseMessage m = await ctx.HttpClient.SendAsync(req))
+                {
+                    ValidateResponse(m, httpVersion);
+                    ValidateContent($"{formData.expected}", await m.Content.ReadAsStringAsync());;
+                }
+            }),
+
+            ("POST Multipart Form Data",
+            async ctx =>
+            {
+                (string expected, MultipartFormDataContent formDataContent) formData = GetMultipartFormDataContent(contentSource, ctx, numParameters);
+                Version httpVersion = ctx.GetRandomVersion(httpVersions);
+
+                using (var req = new HttpRequestMessage(HttpMethod.Post, serverUri) { Version = httpVersion, Content = formData.formDataContent })
+                using (HttpResponseMessage m = await ctx.HttpClient.SendAsync(req))
+                {
+                    ValidateResponse(m, httpVersion);
+                    ValidateContent($"{formData.expected}", await m.Content.ReadAsStringAsync());;
+                }
+            }),
```

Comment:
What differences are you expecting to highlight between these two tests? i.e. what kinds of issues are you expecting to see result from one but not the other?

Reply:
The main purpose of adding this was to test the subtype which sets the content type of the headers. https://github.com/dotnet/corefx/pull/39437/files/7b114272d0640a2d1e1392358b97072dc0caaa71#diff-b1051df593658fad94224d7fe1cbe724R692

But you are right the second one is the superset of the first. I will remove the first.